.github/workflows/ci_push.yml

```diff
@@ -31,7 +31,7 @@ jobs:
 
     - name: flake8
       run: |
-        run_flake8.sh
+        ./run_flake8.sh
```

Comment:
if it doesn't work: chmod u+x ;)

Reply:
I am trying that now... :)

Reply:
hmm at least I don't get it running. Do you have any ideas? :)

Reply:
it should be executable now, all that remains is to install flake8 in .github/workflows/install_deps.

Reply:
🤞

Reply:
Halleluja :D